src/js/components/purchaseOrder/PurchaseOrderListTable.jsx

```diff
@@ -437,13 +438,15 @@ const PurchaseOrderListTable = ({
     }
   };
 
+  const totalAmount = () => `${translate('react.purchaseOrder.totalAmount.label', 'Total amount')}: ${totalPrice.toLocaleString([locale, 'en'])} ${currencyCode}`;
```

Comment:
Do we want to hard-code the locale here?

Reply:
Oh sorry, that's probably a default value in the case that locale is null or undefined.

Reply:
yes, the second argument is a fallback/default locale